routes/project/billing.rb

```diff
@@ -131,7 +131,7 @@ class Clover
           })
         end
 
-        flash["notice"] = "Billing info updated"
+        flash["notice"] = "Payment method added successfully. $#{(preauth_amount / 100).to_i} is authorized on your card for verification purposes. It's canceled already and depending on your bank, it may take up to two weeks to refund the money."
```

Comment:
`preauth_amount = [100, 200, 300, 400, 500].sample`, so it must be integer. `Integer/Integer` returns `Integer`. I don't think there is a need for `to_i`.

```suggestion
        flash["notice"] = "Payment method added successfully. $#{preauth_amount / 100} is authorized on your card for verification purposes. It's canceled already and depending on your bank, it may take up to two weeks to refund the money."
```